spec/abstractops.html

```diff
@@ -82,8 +82,11 @@ <h1>ToDurationLike ( _duration_ )</h1>
       1. If Type(_duration_) is not Object, then
         1. Throw a *TypeError* exception.
       1. For each row of <emu-xref href="#table-temporal-durationlike-properties"></emu-xref>, except the header row, in table order, do
-        1. Let _property_ be the Property value of the current row.
-        1. Let _value_ be ? ToInteger(? Get(_duration_, _property_)).
+        1. If _duration_ has an [[InitializedTemporalDuration]] internal slot, then
+          1. Let _value_ be the value of _duration_'s internal slot whose name is the Internal Slot value of the current row.
```

Comment:
Are we **sure** we want conditional Get bypass on an argument? [`Array.prototype.concat`](https://tc39.es/ecma262/#sec-array.prototype.concat) could have done this but did not, and cases that do seem limited to signatures that _require_ typed input (e.g., [`new DataView`](https://tc39.es/ecma262/#sec-dataview-buffer-byteoffset-bytelength)). I'm not convinced that the benefits of such branches are sufficient to justify the complexity of including them.

Reply:
Indeed, I'm still wondering about code like what I wrote in [this comment](https://github.com/tc39/proposal-temporal/issues/231#issuecomment-577495048)...